example/centos/centos-9-aarch64-image-installer.yaml

```diff
@@ -0,0 +1,576 @@
+otk.version: "1"
+
+otk.define:
+  # XXX these sizes might need to come from an external?
+  size_rootfs: '5368709120'
```

Comment:
Maybe `size_iso_rootfs` or `size_iso_rootfs_image` or something to be a) very clear that this is not the disk image rootfs size but the iso and b) to be close to https://github.com/osbuild/images/blob/v0.89.0/pkg/image/anaconda_tar_installer.go#L157  which seems where this is coming from (maybe even ref it here?)

Reply:
Amended last commit with new names.